src/components/seeds/database/Filters.tsx

```diff
@@ -89,18 +89,17 @@ const getSearchTermFromFilters = (filters: Record<string, SearchNodePayload>): s
 interface Props {
   columns: DatabaseColumn[];
   searchColumns: DatabaseColumn[];
-  preExpFilterColumn: DatabaseColumn;
+  preExpFilterColumns: DatabaseColumn[];
   filters: Record<string, SearchNodePayload>;
   availableValues: FieldValuesPayload;
   allValues: FieldValuesPayload;
   onChange: (filters: Record<string, SearchNodePayload>) => void;
 }
 
 export default function Filters(props: Props): JSX.Element {
```

Comment:
There's a similar component in the v2 inventory area, but I decided to just refactor this one to take an array of filters as opposed to refactoring everything down into a single generic component. 

I made this decision because this thing is pretty tightly coupled to the `Database` component that it is called within, and there is a lot of other stuff going on with searching and filtering because of the "column customization" available in the Accessions area. Let me know if you think that refactor is a must in this case or if this approach is alright.